utils/bboxUtils.js

```diff
@@ -18,7 +18,7 @@ const bbox = (node, applyTransformations) => {
         parseFloat(node.getAttribute('y')) || 0,
         parseFloat(node.getAttribute('width')) || 0,
         parseFloat(node.getAttribute('height')) || 0
-      )
+      ).transform(node.matrixify());
     case 'svg':
```

Comment:
When you do that, you transform the bbox with the transformation matrix of the element. This is not how bbox is defined. Bbox always returns the bounding box in userspace. That means its the coordinates in the transformed space. If you apply the transformation you basically say: I want the bbox in space of the parent element.

Or am I totally wrong here?

Reply:
I'll check with chromium rendering. I have (unpublished) [puppeteer](https://github.com/GoogleChrome/puppeteer) test and I have to convert test to run on node and chromium and compare results.

Reply:
Agree, my fault. Fixed with adbd079